ipc/source/GGCoreIpcClient.cpp

```diff
@@ -0,0 +1,508 @@
+#include <aws/crt/Api.h>
+#include <aws/ipc/GGCoreIpcClient.h>
+
+#include <sstream>
+
+namespace Aws
+{
+    namespace Eventstreamrpc
+    {
+        namespace Ipc
+        {
+            BinaryMessage::BinaryMessage(
+                Crt::Optional<Crt::Vector<uint8_t>> &&message,
```

Comment:
Remove field arguments from the constructor. If you have them there and additional fields are added, it will trivially break existing code when the order of arguments is changed or more are added.

If you want a convenient way to build/initialize objects in a fluent combined statement, and keep a reasonable path for backwards compatibility, you need to generate builders.